mediaengine.go

```diff
@@ -41,6 +42,47 @@ func (m *MediaEngine) RegisterDefaultCodecs() {
 	m.RegisterCodec(NewRTPVP9Codec(DefaultPayloadTypeVP9, 90000))
 }
 
+// PopulateFromSDP finds all codecs in a session description and adds them to a MediaEngine, using dynamic
+// payload types and parameters from the sdp.
+func (m *MediaEngine) PopulateFromSDP(sd sdp.SessionDescription) error {
```

Comment:
Can this take `webrtc.SessionDescription`. I want to avoid having people importing other packages in the happy case and have them run into type issue etc..